junit-jupiter-engine/src/module/org.junit.jupiter.engine/module-info.java

```diff
@@ -14,6 +14,7 @@
 	requires transitive org.junit.jupiter.api;
 	requires transitive org.junit.platform.engine;
 	requires transitive org.opentest4j;
+	requires transitive java.management;
```

Comment:
> Please add `requires java.management;` to ...

As the `java.management` module is only used internally and module `org.junit.jupiter.engine` doesn't "leak" a management type via it's API, there's no need for the `transitive` modifier. Please remove it here and in the integration test.